apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components/Input.stories.mdx

```diff
@@ -0,0 +1,81 @@
+| `fluid`                | n/a                                         |                                                                                                             |
+| `icon`                 | Use `contentBefore` or `contentAfter` slots |                                                                                                             |
+| `iconPosition`         | n/a                                         |                                                                                                             |
+| `inline`               | n/a                                         |                                                                                                             |
+| `input`                | `input`                                     | This is a slot                                                                                              |
+| `inputRef`             | Pass a `ref` to the `input` slot            |                                                                                                             |
+| `inverted`             | `appearance`                                |                                                                                                             |
+| `label`                | Use `Label` component                       | Be sure to associate `Label` with `Input` via `htmlFor`                                                     |
+| `labelPosition`        | n/a                                         |                                                                                                             |
+| `onChange`             | `onChange`                                  |                                                                                                             |
+| `required`             | `required`                                  | This is the native HTML prop                                                                                |
+| `showSuccessIndicator` | n/a                                         | Use `contentBefore` or `contentAfter` slots to insert custom indicators                                     |
+| `type`                 | `type`                                      | Non text types like 'button' and 'checkbox' are not supported. Use `Button` or `Checkbox` component instead |
+| `value`                | `value`                                     | Mutually exclusive with `defaultValue`                                                                      |
+| `variables`            | n/a                                         | Use `FluentProvider` to customize themes                                                                    |
+| `wrapper`              | n/a                                         |                                                                                                             |
```

Comment:
`wrapper` I think in this case would be the `root` slot in the v9 input (or the other way around). Should we note the difference here ?